engarde/decorators.py

```diff
@@ -161,7 +161,19 @@ def wrapper(*operation_args, **operation_kwargs):
         return wrapper
     return decorate
 
-__all__ = ['is_monotonic', 'is_shape', 'none_missing', 'unique_index',
-           'within_range', 'within_set', 'has_dtypes', 'verify', 'verify_all',
-           'verify_any', 'within_n_std']
+
+def is_same_as(df_to_compare, **kwargs):
+    def decorate(func):
+        @wraps(func)
+        def wrapper(*args, **kwargs):
```

Comment:
Ugh, this is a bit complicated. Can you rename the outer `is_same_as` `**kwargs` to `**assert_kwargs` and then pass those to `ck.is_same_as` on line 170? Then the `wrapper` `**kwargs` are for whatever `func` is being decorated (like you have now).